### ISSUE TITLE
Crash when accessing Output Settings

### ISSUE THREAD

**Issue by jcktrbbt** [turtletooth/tahoma2d#516]
**Describe the bug**
The program crashes whenever trying to access the "Output Settings..." window under the "Render" tab. The same happens when trying to use the keyboard shortcut Ctrl+O. 
Other menu items such as "Render", "Preview" and "Preview Settings" seem to function normally. 

**To Reproduce**
List steps to reproduce the behavior.
1. Have working file open/loaded
2. Click on "Render" tab
3. Click on "Output Settings..."

**Device Information:**
 - OS: Windows 10 Home
 - Tahoma2D Version: 1.1 {Oct 29]
 - GPU: Intel HD Graphics 520
 - GPU Driver Version: 26.20.100.8142

**Comment by jeremybullock** [comment]
I can't duplicate this.  Do you have any more info you can share?

**Comment by manongjohn** [comment]
After the crash, please open "Event Viewer" and look in "Windows Logs -> Application" for a crash log for Tahoma2D. There is a small chance the contents of it might help determine what is causing the crash.

**Comment by megaener** [comment]
### I'm having the same exact issue:
List steps to reproduce the behavior.

Have working file open/loaded (even a brand new scene from a brand new project)
Click on "Render" tab
Click on "Output Settings..."

Device Information:
OS: Windows 10 Pro
Tahoma2D Version: 1.1 {Oct 29]
GPU: NVIDIA GeForce 1060
GPU Driver Version: 457.51

**Comment by GlutenBoy** [comment]
I'm also having the same issue.
* Load my project and scene
* Click Render menu
* Than click Render Settings...

it maybe related, but I clean also Clean up settings... it also crashes

Tahoma2D Version 1.2 May 12 2021

**Comment by GlutenBoy** [comment]
Update if I run Tahoma in admin mode (on Windows) I don't get the crash.

**Comment by manongjohn** [comment]
For windows users, it is likely because the `tahomastuff` folder may not have the right permissions, which can sometimes happen when unpacking the zip file and dragging the Tahoma2D folder to certain places.  Not sure quite how that happens, but it can lead to all sorts of issues, including crashing, whenever it tries to either read or write files.

So verify you have the full permissions as follows:

In an explorer window, go to the `Tahoma2D\tahomastuff` folder, right-click it and choose `Properties`.  Click on the `Security` tab. Verify your username is listed in the top box and that it has `Full Control` access to the folder.  If you do not see your username in the list, you will need to add it and grant it full access to the folder.

**Comment by megaener** [comment]
> For windows users, it is likely because the `tahomastuff` folder may not have the right permissions, which can sometimes happen when unpacking the zip file and dragging the Tahoma2D folder to certain places.  Not sure quite how that happens, but it can lead to all sorts of issues, including crashing, whenever it tries to either read or write files.
> 
> 
> 
> So verify you have the full permissions as follows:
> 
> 
> 
> In an explorer window, go to the `Tahoma2D\tahomastuff` folder, right-click it and choose `Properties`.  Click on the `Security` tab. Verify your username is listed in the top box and that it has `Full Control` access to the folder.  If you do not see your username in the list, you will need to add it and grant it full access to the folder.

Good tip thank you. 

I know it's not related but can you have both OT and Tahoma installed at the same time?

Last time I tried that made OT not to launch.

**Comment by manongjohn** [comment]
> I know it's not related but can you have both OT and Tahoma installed at the same time?

Yes you can.  OT may not launch for any number of reasons, including a permission problem on the `OpenToonz stuff` folder.

**Comment by artisteacher** [comment]
If this is a permissions issue, maybe @megaener, @GlutenBoy and @jcktrbbt could share where they placed the Tahoma2D directory, to see if there are any commonalities.
